src/tests/ftest/nvme/io_verification.py

```diff
@@ -48,22 +48,15 @@ def test_nvme_io_verification(self):
         :avocado: tags=daosio,nvme_io_verification
         """
         # Test params
-        tests = self.params.get("ior_sequence", '/run/ior/*')
         processes = self.params.get("np", '/run/ior/*')
         transfer_size = self.params.get("tsize", '/run/ior/transfersize/*/')
         block_size = self.ior_cmd.block_size.value
+        ior_seq_pool_qty = self.params.get("ior_sequence_pool_qty", '/run/pool/*')
 
-        # Loop for every IOR object type
-        for ior_param in tests:
-            # Create and connect to a pool
-            self.add_pool(create=False)
-
-            # update pool sizes
-            self.pool.scm_size.update(ior_param[0])
-            self.pool.nvme_size.update(ior_param[1])
-
-            # Create a pool
-            self.pool.create()
+        # Loop for every pool size
+        for index in range(ior_seq_pool_qty):
+            # Create and connect to a pool with namespace
+            self.add_pool(namespace="pool_{}".format(index))
```

Comment:
```suggestion
            self.add_pool(namespace="/run/pool_{}/*".format(index))
```

Reply:
Thanks for the comment, please see the update.